src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -62,15 +62,20 @@ public PersonCard(Person person, int displayedIndex) {
         phone.setText("\uD83D\uDCDE\t" + person.getPhone().value);
         address.setText("\uD83C\uDFE0\t" + person.getAddress().value);
         email.setText("\uD83D\uDCE7\t" + person.getEmail().value);
+
         // replace birthday and insurance once implemented.
         // insurance.setText("\uD83D\uDCC4\t" + "something something");
-        healthInsurance.setText("\uD83D\uDCDE\t" + (person.getHealthInsurance().getHasInsurance() ? "yes" : "no"));
-        disabilityInsurance.setText("\uD83D\uDCDE\t"
-                + (person.getDisabilityInsurance().getHasInsurance() ? "yes" : "no"));
-        criticalIllnessInsurance.setText("\uD83D\uDCDE\t"
-                + (person.getCriticalIllnessInsurance().getHasInsurance() ? "yes" : "no"));
-        lifeInsurance.setText("\uD83D\uDCDE\t" + (person.getLifeInsurance().getHasInsurance() ? "yes" : "no"));
-        birthday.setText("\uD83C\uDF82\t" + person.getBirthday().toString());
+
+        healthInsurance.setText("Health               "
+                + (person.getHealthInsurance().getHasInsurance() ? "\u2705" : "\u274e"));
+        disabilityInsurance.setText("Disability           "
+                + (person.getDisabilityInsurance().getHasInsurance() ? "\u2705" : "\u274e"));
```

Comment:
Do you think this kind of violates the "don't talk to strangers" rule? We access the DisabilityInsurance object from the person object and operate on that object? 🤔